TickingHeartRate/runTickingHeartRateReplay.py

```diff
@@ -1,17 +1,15 @@
-from deephaven import csv
+from deephaven import read_csv
 from deephaven import DynamicTableWriter, Types as dht
 from deephaven import DateTimeUtils
 import pathlib
 import time
 import threading
 
-import sys
-
 # Max number of csv files to pull in
 csv_files=500
 
-# Setup deephaven tables to hold Heart Rate results
-column_names = ["Timestamp", "Heart Rate"]
+# Setup deephaven tables to hold Heart_rate results
+column_names = ["Timestamp", "Heart_rate"]
```

Comment:
```suggestion
column_names = ["Timestamp", "HeartRate"]
```
Naming convention for column names is camel case